tests/unit_test/linux/logging-stack/logging.cmake

```diff
@@ -0,0 +1,3 @@
+# Configuration for logging.
+set( LOGGING_INCLUDE_DIRS
```

Comment:
This file isn't required in the repository as the unit test CMakeLists.txt will directly include the `tests/unit_test/linux/logging_stack` directory path for accessing the `logging_levels.h` and `logging_stack.h` files

Reply:
removed in f5095e0ad5300ba6f97a03cd3d1622b525afc0b9